### DISCUSSION TITLE
Voxelab Aquila support for PROUI_EX?

### DISCUSSION THREAD

**Discussion by classicrocker883** [mriscoc/Ender3V2S1#1026] [General]
since the August update, `BOARD_VOXELAB_AQUILA` isn't supported as it is a GD32 chip.

even though it runs Maple and it's depreciating, are there plans to bring it back? because these GD32 will not boot using standard STM32F103RC_creality. they so far only can use the _Maple environment

**Comment by mriscoc** [comment]
There are many reports from users with 4.2.2 board and GD32 SoC, where the normal firmware works for them, it is a bit tricky and depends on a good SD card and format.

**Reply by classicrocker883** [reply]
maybe that is the issue. the SD cards that came standard with the printers may not be the best.
